### PR TITLE
Categories within products

### PR DESCRIPTION
**Pull request by starkshaw**

### CODE REVIEW

**Comment by danielpaul** [comment]
@LevinGeorge assign it to me once you have the front end category select working and I can then review and merge.

**Comment by LevinGeorge** [comment]
@danielpaul :+1:

File: app/admin/category.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    id_column
+    column :title
+    actions
+  end
+
+  filter :title
+
+  form do |f|
+    f.inputs "Categories" do
+      f.input :title
```
Comment (danielpaul) [inline app/admin/category.rb:17]: We need to add description column for the categories

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@ def show
   def new
     @product = Product.new
     add_breadcrumb "New Product", :new_shop_product_path
+    @category = Category.all
```
Comment (danielpaul) [inline app/controllers/products_controller.rb]: You are getting categories so better to name collections in plural.

File: app/admin/category.rb (hunk 1 above)
Comment (LevinGeorge) [inline app/admin/category.rb:17]: Oh yeah!

**Comment by danielpaul** [comment]
@LevinGeorge let's get the description for the category column in and it's ready to be merged then :)

File: app/admin/category.rb (hunk 1 above)
Comment (LevinGeorge) [inline app/admin/category.rb:17]: @danielpaul Done. Added a description field for active admin.

**Comment by LevinGeorge** [comment]
@danielpaul Somehow it's not saving the chosen category..

**Pull request merged**